### PR TITLE
pvr.dvblink addon v4.5.0

### PR DESCRIPTION
**Pull request by dvblogic**
Updated to PVR API v5.8.0

### CODE REVIEW

**Comment by dvblogic** [comment]
@ksooo Is it ok for me to merge or I need review input from someone from the Kodi team?

File: pvr.dvblink/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[B]Version 4.5.0[/B]
+Updated to PVR API v5.8.0
```
Comment (ksooo) [inline pvr.dvblink/changelog.txt:2]: Shouldn't you mention that TVMosaic support was added?

File: src/RecordingStreamer.cpp [hunk 2]
```diff
@@ -100,18 +100,26 @@ long long RecordingStreamer::SeekRecordedStream(long long iPosition, int iWhence
...
 
-long long RecordingStreamer::LengthRecordedStream(void)
+PVR_ERROR RecordingStreamer::GetStreamTimes(PVR_STREAM_TIMES* stream_times)
 {
-  return recording_size_;
+  if (stream_times != NULL)
+  {
+    stream_times->startTime = 0;
+    stream_times->ptsStart = 0;
+    stream_times->ptsBegin = 0;
```
Comment (ksooo) [inline src/RecordingStreamer.cpp:114]: This implementation implies that your add-on does not support time shifting. Is this correct? If not, you must supply a real implementation.

Comment (ksooo) [reply src/RecordingStreamer.cpp:114]: Nvm, just realised that this implementation is for recordings only

**Review by ksooo — COMMENTED** [review]
Isn't TVMosaic worth being mentioned in the change log?

File: pvr.dvblink/changelog.txt (hunk 1 above)
Comment (dvblogic) [reply pvr.dvblink/changelog.txt:2]: Added and merged. Thanks!

**Pull request merged**